mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -55,6 +55,13 @@ wikiRatingEditorHash = (table) ->
       hash[key] = tr.find(".metric-weight input").val()
   hash
 
+toEqualize = (table) -> 
+  val = (100 / (table.find("tbody tr").length - 1)).toFixed(2)
```

Comment:
this pattern: `table.find("tbody tr")` is repeated.  I think you should make a separate function for it called something like `variableMetricRows`.